CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Same changes as in `@mui/x-data-grid-pro@7.22.0`, plus:
 
 #### `@mui/x-date-pickers@7.22.0`
 
+- [pickers] Fix `DateCalendar` timezone management (#15119) @LukasTy
```

Comment:
@LukasTy Does it makes sense to be highlighted?

Reply:
It's technically a bugfix, I'm not sure we usually do it and should. 🤔 
WDYT?

Reply:
The number of issues it closes means it's solving quite a few problems, might be nice to highlight, feel free to proceed as you may wish.

Reply:
Good point, but I'm good without an explicit highlight. 🤔 👌 
No strong preference here. If you feel like it would make a lot of sense—feel free to include it. 👍